### PR TITLE
provision fixes

### PR DESCRIPTION
**Pull request by sofuture**
hook updates, moved around dev targets

### CODE REVIEW

File: dev/bootstrap.yaml [hunk 1]
```diff
@@ -7,4 +7,21 @@ spec:
...
+      volumeMounts:
+        - mountPath: /var/lib/gravity/resources
+          name: resources
+        - mountPath: /etc/ssl/certs
+          name: certs
+        - mountPath: /usr/local/bin/kubectl
+          name: kubectl
+  volumes:
+  - hostPath:
+      path: /var/lib/gravity/local/packages/unpacked/gravitational.io/stolon-app/0.0.4/resources
```
Comment (sofuture) [inline dev/bootstrap.yaml]: This is fragile and probably not ideal going forward.

Comment (r0mant) [inline dev/bootstrap.yaml]: Umm, it is mounted automatically (by gravity) inside the hook container. This is for the install hook, right?

Comment (r0mant) [inline dev/bootstrap.yaml]: I mean you don't need to manually mount it here, it will be inside the hook container automatically.

Comment (sofuture) [inline dev/bootstrap.yaml]: This is for creating the app in a dev workflow (i.e. redeploying with `kubectl delete` + `kubectl create`), so in that case, no.

**Pull request merged**